src/components/Views/EmptyDataView.jsx

```diff
@@ -1,17 +1,31 @@
 import React from 'react'
 
+import { useAppLinkWithStoreFallback, useClient } from 'cozy-client'
+import AppLinker from 'cozy-ui/transpiled/react/AppLinker'
 import Button from 'cozy-ui/transpiled/react/Buttons'
 import Empty from 'cozy-ui/transpiled/react/Empty'
+import { useI18n } from 'cozy-ui/transpiled/react/providers/I18n'
 
-import ImportSchoolIcon from '../../assets/illustrations/import-school.svg'
 import PronoteIcon from '../../assets/brands/pronote.svg'
-import EcoleDirecteIcon from '../../assets/brands/ecoledirecte.svg'
-
-import { useI18n } from 'cozy-ui/transpiled/react/providers/I18n'
+import ImportSchoolIcon from '../../assets/illustrations/import-school.svg'
 
 export const EmptyDataView = () => {
   const { t } = useI18n()
 
+  const client = useClient()
+
+  const { url: pronoteKonnectorUrl } = useAppLinkWithStoreFallback(
```

Comment:
The `useAppLinkWithStoreFallback` only works for apps since it checks their existence in the io.cozy.apps : https://github.com/cozy/cozy-client/blob/6e00bf0329e8a8cf406d718782e184885a016b35/packages/cozy-client/src/hooks/useAppLinkWithStoreFallback.jsx#L17
Therefore, even if the konnector is installed, the given URL will probably be the store link rather than the actual konnector link. So we should extend the useAppLinkWithStoreFallback to deal with konnector, or create a new hook, e.g. `useKonnectorLinkWithStoreFallback`